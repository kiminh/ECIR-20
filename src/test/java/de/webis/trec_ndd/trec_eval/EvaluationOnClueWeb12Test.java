package de.webis.trec_ndd.trec_eval;

import org.approvaltests.Approvals;
import org.junit.Ignore;
import org.junit.Test;

import de.webis.trec_ndd.trec_collections.SharedTask.TrecSharedTask;

public class EvaluationOnClueWeb12Test {
	
	@Test
	public void approveBestERR_10RunFromTable2From2013() {
		// clustmrfaf 0.175
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2013, "clustmrfaf"));
	}
	
	@Test
	public void approveWorstMapRunFromTable2From2013() {
		// dlde 0.008
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2013, "dlde"));
	}
	
	@Test
	public void approveuogTrAIwLmbFromTable2From2013() {
		// uogTrAIwLmb 0.151
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2013, "uogTrAIwLmb"));
	}
	
	@Test
	public void approveXXXRunFromTable2From2013() {
		// udemQlm1lFbWiki
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2013, "udemQlm1lFbWiki"));
	}
	
	@Test
	@Ignore
	public void approveBestERR_IA_20RunFromTable2From2014() {
		// UDInfoWebLES 0.688
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2014, "UDInfoWebLES"));
	}
	
	@Test
	public void approveWorstERR_IA_20RunFromTable2From2014() {
		// SNUMedinfo12 0.531
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2014, "SNUMedinfo12"));
	}
	
	@Test
	public void approveTerranFromTable2From2014() {
		// Terran 0.578
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2014, "Terran"));
	}
	
	@Test
	public void approveudelwebisWt14axMaxRunFromTable2From2014() {
		// webisWt14axMax 0.589
		Approvals.verifyAsJson(EvaluationOnClueWeb09Test.evaluateAllOfficialMeasuresForRun(TrecSharedTask.WEB_2014, "webisWt14axMax"));
	}
}
