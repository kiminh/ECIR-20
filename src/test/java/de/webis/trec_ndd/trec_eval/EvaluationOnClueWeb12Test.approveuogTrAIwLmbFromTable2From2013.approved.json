{
  "uogTrAIwLmb": {
    "ERR_20": "0.160",
    "ERR_10": "0.151",
    "NDCG_CUT_10": "0.247",
    "NRBP": "0.498",
    "ALPHA_NDCG_CUT_20": "0.637",
    "NDCG_CUT_20": "0.259",
    "ALPHA_NDCG_CUT_10": "0.606",
    "ERR_IA_20": "0.548",
    "ERR_IA_10": "0.539"
  }
}